src/tests/multihost/alltests/test_hosts.py

```diff
@@ -9,13 +15,15 @@
 class TestHostMaps(object):
     """
     SSSD NSS should support host map
-    @bugzilla:
-    https://bugzilla.redhat.com/show_bug.cgi?id=1340908
+    :bugzilla:
+     https://bugzilla.redhat.com/show_bug.cgi?id=1340908
     """
     @pytest.mark.tier1
     def test_001_ldap_iphost_search_base(self, multihost):
         """
-        :Title:hosts: Test ldap_iphost_search_base field
+        :title: hosts: Test ldap_iphost_search_base field
+        :id: a5b82f87-e4f2-49ce-bfc2-e23f5020219d
+        :customerscenario: True
```

Comment:
Entire test script does not have any :steps: and :expectedresults:

Reply:
Change is only to update docstrings for auto import